MultiThreading.Task3.MatrixMultiplier.Tests/MultiplierTest.cs

```diff
@@ -18,8 +19,48 @@ public void MultiplyMatrix3On3Test()
         [TestMethod]
         public void ParallelEfficiencyTest()
         {
-            // todo: implement a test method to check the size of the matrix which makes parallel multiplication more effective than
-            // todo: the regular one
+            var efficientSize = 0;
+            Stopwatch stopwatch = Stopwatch.StartNew();
```

Comment:
`Stopwatch.StartNew` creates a new instance and immediately starts time measurement, I think you want just `new Stopwatch()` here